src/wix/wix.cmd

```diff
@@ -24,9 +24,9 @@ msbuild -t:Restore wix.sln -p:Configuration=%_C% -nologo -m -warnaserror -bl:%_L
 
 
 :: Build
-msbuild wixnative\wixnative_t.proj -p:Configuration=%_C% -nologo -m -warnaserror -bl:%_L%\wixnative_build.binlog || exit /b
```

Comment:
I thought this had been around for a long while and was not causing problems. It was only publish that seemed to have -m problems, no?

Reply:
Sorry, didn't mean to commit that. But yes, I'm having consistent build breaks with files-in-use errors.